### DISCUSSION TITLE
Airflow 2.5.3 does not run Scheduled DAGs, deployed via Helm Chart

### DISCUSSION THREAD

**Discussion by ghost** [apache/airflow#31707] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

Currently Airflow (v2.5.3) does not execute scheduled DAG. We deployed the latest Helm Chart 1.9 and we are experience some Issues.

We have noticed that the the last run was after the latest run, for example: 
Last run was timed for 2023-06-01, 11:47:30 while the next run was scheduled for 2023-06-01, 11:27:59.

When we reload the page the timestamp for the Next Run updates to the round about 10 sec. behind the current time.

We also noticed that when we check the pop up field for next run, when in the few of the DAG details, Airflow claims that the next run is scheduled for a few seconds ago.

<img width="421" alt="Screenshot 2023-06-02 at 14 09 35" src="https://github.com/apache/airflow/assets/68283314/ebf69af7-57f4-488d-91c4-04a1cdb50f4a">

DAG can be triggered manually.

### What you think should happen instead

Next Schedule should be in the future and DAG should run over night.

### How to reproduce

Sadly I don't know.

### Operating System

Ubuntu 22.04

### Deployment

Official Apache Airflow Helm Chart

### Deployment details

RKE2 Deployment on Ubuntu 22.04
Helm Chart version 1.9
Externe Postgres Database

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
Sadly, without some details or logs on how to reproduce it or what kind of special circumstances caused it, we are not able to help.

Since the installation works out-of-the-box for us (we have tests in our CI that run deployment and DAG running) reporting an issue without anyl extra information, log analysis, details that would bring us closer to succesfull diagnosis, it's just a beginning of discussion, not an actionable issue.

We need more details - resulting from an effort to analyse the problem, loking at the logs, analysing warnings and problems reported there to be able to help you to solve your problem. This is an open-source project, if things are working for us in tests (they do), it's on  you to provide some extra information that will help us to help you.
